### ISSUE TITLE
not showing preview marker for blog on homepage

### ISSUE THREAD

**Issue by Edwin-Luijten** [redbaron76/PongoCMS-Laravel-cms-bundle#8]
When adding a new element with the preview marker:
[$PREVIEW[{"source":"blogs", "n":"3", "tpl":"latest_blogs"}]]

It only shows the element div but not the blogs. Al tough it works on the blog page.

**Comment by redbaron76** [comment]
Does "latest_blogs" view file exist in theme's partial/markers?

**Comment by Edwin-Luijten** [comment]
Yes it exists, and the code inside is identical as the preview view file (because that file seems to work)

**Comment by redbaron76** [comment]
Well.. I don't know why... 
Using default template is it working?

**Comment by Edwin-Luijten** [comment]
Tried the default template to, I'll be checking how and what's happening and post back if I fixed it.

I also created the element in the demo on your site, you'll see it at the home page, right column.
